docker-compose.yml

```diff
@@ -0,0 +1,44 @@
+version: '3'
+
+services:
+
+  # assumes imgsrv-sample-data has been checked out next to "imgsrv"
+  dev:
+    build: .
+    volumes:
+      - .:/htapps/babel/imgsrv
+      - "../imgsrv-sample-data/sdr1:/sdr1"
+      - "../imgsrv-sample-data/etc:/htapps/babel/etc"
+      - "../imgsrv-sample-data/watermarks:/htapps/babel/watermarks"
+      - "../imgsrv-sample-data:/tmp/imgsrv-sample-data"
+    environment:
+      - SDRROOT=/htapps/babel
```

Comment:
Not sure I understand why this is in both docker-compose.yml and the Dockerfile?

Reply:
I'll confess to being unclear which ENV setup is available when in the container lifecycle. Some ENV ended up in Dockerfile when I was trying to `RUN bin/startup_imgsrv`, but that approach had other odd consequences. I'll see which can be cleared up.